content/docs/tutorials/dess/gcp/4-prep-network-access/index.md

```diff
@@ -62,7 +62,7 @@ If everything goes well you should see following in your domain dashboard:
 
 Next step is to update Google Name servers. You can follow Googles guide - step 5.
 
-https://cloud.google.com/dns/docs/tutorials/create-domain-tutorial#register-domain
+{{< a href="https://cloud.google.com/dns/docs/tutorials/create-domain-tutorial#register-domain" target="_blank">}}Link to tutorial{{< /a >}}
```

Comment:
We tend to prefer that a markdown link is used, or an appropriate hugo shortcode if one already exists in the theme.